grafana-formula/grafana-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jan 22 08:31:16 UTC 2021 - Witek Bedyk <witold.bedyk@suse.com>
+
+- Version 0.4.0
+- Add SUSE 15.3 to supported versions
```

Comment:
`SUSE Linux Enterprise 15 SP3` instead of `SUSE 15.3`,  maybe?

Reply:
Added `openSUSE` as well.